testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_delta_sync.py

```diff
@@ -712,10 +712,8 @@ def test_delta_sync_larger_than_doc(params_from_base_test_setup, num_of_docs, re
 @pytest.mark.syncgateway
 @pytest.mark.replication
 @pytest.mark.parametrize("num_of_docs, replication_type, file_attachment, continuous", [
-    (10, "pull", None, True),
-    (10, "pull", "sample_text.txt", True),
-    (1, "push", "golden_gate_large.jpg", True),
-    (10, "push", None, True)
+    (10, "pushAndPull", None, True),
+    (10, "pull", "sample_text.txt", True)
```

Comment:
I can understand push pull are combined, is push with attachment no longer needed? just curious.

Reply:
this test is specifically for community edition. For regular tests i.e test run with EE/EE already covers with all combinations. For CE delta sync tests, we do not need duplicate testing with all combinations. Just with one attachment and one without attachment is good enough

Reply:
sounds good